app/mockServices/mockAlertMessage.service.ts

```diff
@@ -0,0 +1,46 @@
+interface IAlertMessageService {
+  addAlert(alert: any) : any;
+  getAlerts() : any;
+  clearAlerts(): any;
+  isAlertPermanentlyHidden(alertID: any, namespace: any): any;
+  permanentlyHideAlert(alertID: any, namespace: any): any;
+}
+
+
+export class AlertMessageService implements IAlertMessageService {
```

Comment:
Same issue here. The idea is not to duplicate the real implementation just to have a basic implementation that allows for testing without using the common services.